components/tcf/secondLayer/src/index.js

```diff
@@ -150,16 +91,20 @@ export default function TcfSecondLayer({
     ADEVINTA_COLLECTED_CONSENTS.specialFeatures.forEach(
```

Comment:
This evaluation shouldn't be done here.

Reply:
I think it's not easy at this moment, will check but not sure if it can be done without a bigger refactor

Reply:
done, I've refactored all code to provide the consents from the outside, marking it a TECH-DEBT for a further refactor when integrated with the sites. The scope of the consent now will be enabled to be provided from the integrable UI component (but now, by default it is the scope that was hardcoded yet).